polls/templatetags/poll_votings.py

```diff
@@ -12,8 +12,9 @@
                         takes_context=True)
 def poll_page(context, pk=None, page=None):
     context = copy(context)
-    polls = Question.objects.filter(page=page.id)
-    context.update({
-        'questions': polls
-    })
+    if page:
+        questions = Question.objects.live().child_of(page)
+        context.update({
+            'questions': questions
+        })
```

Comment:
Wouldn't we rather just return an empty list instead of not having the `questions` variable?